src/util/common.jl

```diff
@@ -0,0 +1,308 @@
+                if nw_id != "0" #not items are repaired in "0", do not check in previous net for a change
+                    if comp[status_key] != _PM.pm_component_status_inactive[comp_type] &&  # if comp is active
+                        network["nw"]["$(parse(Int,nw_id)-1)"][comp_type][comp_id][status_key] == _PM.pm_component_status_inactive[comp_type] # if comp was previously inactive
+                        push!(repairs[nw_id], (comp_type,comp_id))
+                    end
+                end
+            end
+        end
+    end
+    return repairs
+end
+
+
+"count repairs in each period of the solution data"
+function _count_component_repairs(solution)
+    repairs = _get_component_repairs(solution)
+    repair_count = Dict{String,Int}(nw=>length(nw_repairs) for (nw, nw_repairs) in repairs )
+    return repair_count
+end
+
+
+"count the cumulative repairs in each period of the solution data"
+function _count_cumulative_component_repairs(solution)
+    repair_count = _count_component_repairs(solution)
+
+    cumulative_count = Dict{String,Int}(nw_id=>0 for nw_id in keys(repair_count))
+    end_nw_id = maximum(parse.(Int,keys(repair_count)))
+
+    for (nw_id, count) in repair_count
+        for id in parse(Int,nw_id):end_nw_id
+            cumulative_count["$(id)"]+= count
+        end
+    end
+    return cumulative_count
+end
+
+
+"""
+Update sol_1 dictionary with sol_2 dictionary. `termination_status`, `primal_status`, and `dual_status`
+values are the maximum of the MOI status code. `solvetime`,`objective`, and `objective_lb` are accumulated. The
+`solution` dictionary is merged using `PowerModels.update_data(sol_1["solution"],sol_2["solution"])`.
+"""
+function merge_solution!(sol_1, sol_2)
+    Memento.info(_PM._LOGGER, "networks $(keys(sol_2["solution"]["nw"])) finished with status $(sol_2["termination_status"])")
+
+    sol_1["termination_status"] = max(sol_1["termination_status"],sol_2["termination_status"])
+    sol_1["primal_status"] = max(sol_1["primal_status"],sol_2["primal_status"])
+    sol_1["dual_status"] = max(sol_1["dual_status"],sol_2["dual_status"])
+    sol_1["solve_time"] += sol_2["solve_time"]
+    sol_1["objective"] += sol_2["objective"]
+    sol_1["objective_lb"] += sol_2["objective_lb"]
+end
+
+
+"Transforms a single network into a multinetwork with several deepcopies of the original network. Indexed from 0."
+function _simple_replicate_restoration_network(sn_data::Dict{String,<:Any}; count::Int=1, global_keys::Set{String}=Set{String}())
+    # differs from replicate_restoration_network by not by not reducing count to the total number of repairs
+    return _simple_replicate_restoration_network(sn_data, count, union(global_keys, _PM._pm_global_keys))
+end
+
+
+"Transforms a single network into a multinetwork with several deepcopies of the original network. Indexed from 0."
+function _simple_replicate_restoration_network(sn_data::Dict{String,<:Any}, count::Int, global_keys::Set{String})
```

Comment:
Why `_simple_replicate_restoration_network` and not `_replicate_restoration_network`?

Reply:
This variant is only used in RAD to create a single repair per period restoration multinetwork, rather than calculating the repairs per period given damaged components and a number of repair periods to use.

The function name was changed to `_single_repair_restoration_network`.  Does this clarify its use?